dangerzone/isolation_provider/base.py

```diff
@@ -93,8 +103,42 @@ def convert(
             self.print_progress(document, True, str(e), 0)
             document.mark_as_failed()
 
-    def doc_to_pixels(
-        self, document: Document, tempdir: str, p: subprocess.Popen
+    def _pixels_to_pdf(
```

Comment:
I wanted to change the name, because this method's semantics are different from the previous `pixels_to_pdf` method. It basically creates a PDF for a single page.

I guess it's more accurate to rename it to `pixels_to_pdf_page()`.

Reply:
Done in 8884cb8